### DISCUSSION TITLE
Synology install not importing files

### DISCUSSION THREAD

**Discussion by mikelagase** [photoprism/photoprism#4363] [General]
Hi,

I successfully installed PhotoPrism in docker container on Synology and when indexing files it shows that all of my folders in my photo directory are indeed indexed as I see it logging an entry for each folder but it seems that nothing is imported for whatever reason. 

The install method I am using is below. 

docker run -d --name=PhotoPrism \
-p 2342:2342 \
-e PHOTOPRISM_UPLOAD_NSFW=true \
-e PHOTOPRISM_UID=1026 \
-e PHOTOPRISM_GID=100 \
-e TZ=America/Los_Angeles \
-e PHOTOPRISM_ADMIN_USER=photoprism \
-e PHOTOPRISM_ADMIN_PASSWORD=xxxxxx \
-v /volume1/docker/photoprism:/photoprism/storage \
-v /volume1/photo:/photoprism/originals \
--security-opt seccomp=unconfined \
--security-opt apparmor=unconfined \
photoprism/photoprism

I believe I have this setup correctly based on the documentation but cannot figure out why no photos are being shown in the application. I originally had some permissions issues but subsequent installs and indexing no longer shows any permission denied issues. 

Since it is indeed scanning all folders but importing 0, I believe permissions is set correctly. 

Any ideas?

**Comment by graciousgrey** [comment]
Hi,

Possible reasons for missing images can be found in our troubleshooting checklist. I recommend going through it step by step.

- https://docs.photoprism.app/getting-started/troubleshooting/#missing-pictures

You may also obtain additional information by switching on trace or debug logs and observing the logs when indexing a folder.

- https://docs.photoprism.app/getting-started/troubleshooting/logs/#__tabbed_1_3

**Reply by mikelagase** [reply]
I've already been through that list to no avail. I tried a single folder to index from the import list and it finds 0 files. 

2024-07-01 04:59:32 INFO indexing completed in 9 s
2024-07-01 04:59:32 INFO index: updated 0 files [9.197341203s]
2024-07-01 04:59:32 INFO index: found no new or modified files
2024-07-01 04:59:32 INFO classify: loading labels from labels.txt
2024-07-01 04:59:23 INFO classify: loading nasnet
2024-07-01 04:59:23 INFO indexing files in '2017 pics'

Trying to enable debug logging in the UI and restarting the application to see if that shows anything

**Comment by mikelagase** [comment]
Weird, enable debug logging doesn't emit any other type of data on the Logs tab if I do a complete rescan again. scratching head...

**Reply by graciousgrey** [reply]
You could set the log level to trace mode and watch the docker logs in a terminal instead of the UI. There might be more logs visible as not all logs are shown in the web ui due to security reasons. In case no files appear at all I would assume that there is a problem with file permissions.

**Reply by mikelagase** [reply]
Trace logs don't show any permissions issues either. It knows there are 47 folders there but it doesn't index any files. I've checked permissions and I believe the permissions are correct. 

time="2024-07-04T06:18:37-07:00" level=info msg="indexing originals..."                                                                                          
time="2024-07-04T06:18:37-07:00" level=info msg="classify: loading nasnet"                                                                                       
time="2024-07-04T06:18:47-07:00" level=info msg="classify: loading labels from labels.txt"                                                                       
time="2024-07-04T06:18:47-07:00" level=info msg="index: found no new or modified files"                                                                          
time="2024-07-04T06:18:47-07:00" level=info msg="index: updated 0 files [9.745328054s]"                                                                          
time="2024-07-04T06:18:47-07:00" level=debug msg="covers: updated 47 folders [66.393914ms]"                                                                      
time="2024-07-04T06:18:47-07:00" level=debug msg="removed folder:/:true:false from cache"                                                                        
time="2024-07-04T06:18:47-07:00" level=debug msg="purge: updated 0 files [681.862µs]"                                                                           
time="2024-07-04T06:18:47-07:00" level=debug msg="purge: updated 0 duplicates [275.26µs]"                                                                       
time="2024-07-04T06:18:47-07:00" level=debug msg="purge: updated 0 photos [754.295µs]"                                                                          
time="2024-07-04T06:18:47-07:00" level=debug msg="index: found no photos without primary file [995.107µs]"                                                      
time="2024-07-04T06:18:47-07:00" level=debug msg="index: found no orphan files [368.125µs]"                                                                     
time="2024-07-04T06:18:47-07:00" level=debug msg="search: updated index [429.24µs]"                                                                             
time="2024-07-04T06:18:47-07:00" level=debug msg="index: updating counts"                                                                                        
time="2024-07-04T06:18:47-07:00" level=debug msg="counts: updated 1 place [383.975µs]"                                                                          
time="2024-07-04T06:18:47-07:00" level=debug msg="counts: updated 0 subjects [939.506µs]"                                                                       
time="2024-07-04T06:18:47-07:00" level=debug msg="counts: updated 0 labels [663.806µs]"                                                                         
time="2024-07-04T06:18:47-07:00" level=debug msg="index: updating covers"                                                                                        
time="2024-07-04T06:18:47-07:00" level=debug msg="covers: updated 0 albums [610.841µs]"                                                                         
time="2024-07-04T06:18:47-07:00" level=debug msg="covers: updated 47 folders [8.758714ms]"                                                                       
time="2024-07-04T06:18:47-07:00" level=debug msg="covers: updated 0 months [627.243µs]"                                                                         
time="2024-07-04T06:18:47-07:00" level=debug msg="covers: updated 0 labels [1.087943ms]"                                                                         
time="2024-07-04T06:18:47-07:00" level=debug msg="covers: updated 0 subjects [533.746µs]"                                                                       
time="2024-07-04T06:18:47-07:00" level=debug msg="purge: updated index [18.618113ms]"                                                                            
time="2024-07-04T06:18:47-07:00" level=debug msg="moments: analyzing 0 photos and 0 videos, with threshold 3"                                                    
time="2024-07-04T06:18:47-07:00" level=debug msg="moments: not enough files"                                                                                     
time="2024-07-04T06:18:47-07:00" level=info msg="indexing completed in 9 s"

**Reply by lastzero** [reply]
Have you considered using Docker Compose instead of manually running the Docker command with a long list of parameters? That would be easier for you to debug and easier for us to help:
- https://docs.photoprism.app/getting-started/docker-compose/

**Reply by mikelagase** [reply]
Well, I created a new shared folder and dumped in a few pictures in the new folder and that successfully indexed files. It may be some type of file in these folders that are causing this issue so going to try to debug this a little more to see what is going on by using a new shared folder and dropping in the folders with my pics/videos in to them to see where it starts failing.

**Reply by lastzero** [reply]
Remove the `.ppignore` and `.ppstorage` files if they have been added to your originals folder, e.g. due to a misconfiguration:
- https://github.com/photoprism/photoprism/issues/1642

**Reply by mikelagase** [reply]
Those files are being added to my docker configuration directory, not the originals folder. Restarting the container creates the files again in the config directory. I'm going to create another shared folder and start moving my files there to see what happens as a test folder worked.

**Comment by mikelagase** [comment]
What seems to have fixed it is I was using the default /photo shared folder that Synology Photos uses for shared spaces. After I clicked convert to Windows ACL on this shared folder it is now indexing all of my photos/videos now after doing a rescan.  I'll see what everything looks like after indexing is complete but seems I am in better shape now as it is finding the files as it should now.

**Reply by mikelagase** [reply]
Everything is still going good except for the issue described in https://github.com/photoprism/photoprism/issues/4339 which is allowing my indexing to proceed further after implementing the configuration change.
